src/components/UserDashboard/UserCard.js

```diff
@@ -40,8 +40,8 @@ export default function UserCard(props) {
   return (
     <div className="user-card">
       {<Dropdown.Button overlay={menu}></Dropdown.Button>}
-      {user.userid}, {user.username}, {user.firstname},{' '}
-      {user.roles[0].role.name}
+      {console.log(user)}
```

Comment:
Could you get rid of this `console.log` before we merge this, please? Just remove locally and re-commit and push on this same branch and mark this convo as resolved, and we should be good to go.

Reply:
Agreed on this front, everything looks good otherwise!